pycytominer/cyto_utils/cells.py

```diff
@@ -312,11 +343,28 @@ def aggregate_compartment(
         object_df = aggregate(
             population_df=population_df,
             strata=self.strata,
+            compute_object_count=compute_counts,
             operation=self.aggregation_operation,
             subset_data_df=self.subset_data_df,
-            **aggregate_args
+            object_feature=self.object_feature,
+            **aggregate_args,
         )
 
+        if compute_counts:
+            fields_count_df = self.image_df.loc[
+                :, self.strata + [self.fields_of_view_feature]
```

Comment:
should we also check the feature union for this?

Reply:
👍 https://github.com/cytomining/pycytominer/pull/129/commits/e759d5902e162f645e1e5e936d280da7022315fc